lib/rails/generators/alchemy/install/files/application.html.erb

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%= @page.language_code %>">
```

Comment:
Yes. Let's keep it short. If one wants to change that, it's now easy to change 👍